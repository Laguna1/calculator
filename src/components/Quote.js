import React from 'react';
import NavBar from './Navbar';

const Quote = () => (
  <>
    <NavBar />
    <h1>“Experience is the name everyone gives to their mistakes.”</h1>
  </>
);

export default Quote;
