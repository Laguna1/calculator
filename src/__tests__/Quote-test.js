import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Quote Page', () => {
  it('renders correctly', () => {
    const page = renderer.create(<Quote />);
    expect(page).toMatchSnapshot();
  });
});
