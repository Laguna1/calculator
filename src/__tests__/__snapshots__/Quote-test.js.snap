// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Page renders correctly 1`] = `
<div
  className="quote-page"
>
  <h1>
    “Experience is the name everyone gives to their mistakes.” - Unknown Author
  </h1>
</div>
`;
